samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js

```diff
@@ -0,0 +1,166 @@
+    });
+  };
+
+  // Bot's submit handler for task module submit.
+  async handleTeamsTaskModuleSubmit(context, taskModuleRequest) {
+    var review = JSON.stringify(taskModuleRequest.data);
+    var reply = JSON.parse(review);
+    await context.sendActivity(context.activity.from.name + " : " + "**" + reply.myValue + "**" + " for " + "'" + reply.title + "'")
+  }
+
+  // Custom method for sending targeted meeting notifications.
+  async targetedNotification(context, meetingId, selectedMembers) {
+    var serviceUrl = context.activity.serviceUrl;
+    const credentials = new MicrosoftAppCredentials(process.env.MicrosoftAppId, process.env.MicrosoftAppPassword);
+    const botToken = await credentials.getToken();
+
+    // Notification payload for meeting target notification API.
+    let notificationInformation = {
+      type: "targetedMeetingNotification",
+      value: {
+        recipients: selectedMembers,
+        surfaces: [
+          {
+            surface: "meetingStage",
+            contentType: "task",
+            content: {
+              value: {
+                height: "300",
+                width: "400",
+                title: "Targeted meeting Notification",
+                url: `${this.baseUrl}`
+              }
+            }
+          }
+        ]
+      }
+    }
+
+    try {
+      //Make the post http call for sending targeted notifications.
+      var response = await axios.post(`${serviceUrl}/v1/meetings/${meetingId}/notification`, notificationInformation, {
+        headers: {
+          "accept": "application/json",
+          "contentType": 'application/json',
+          "authorization": "bearer " + botToken
+        }
+      });
+    } catch (exception) {
+      console.log(exception);
+    }
+  }
+
+  // Configure and send meeting content bubble.
+  async contentBubble(context) {
+    const replyActivity = MessageFactory.text("**Please provide your valuable feedback**");
+    replyActivity.channelData = {
+      onBehalfOf: [
+        {
+          itemId: 0,
+          mentionType: 'person',
+          mri: context.activity.from.id,
+          displayname: context.activity.from.name
+        }
+      ],
+      notification: {
+        alertInMeeting: true,
+        externalResourceUrl: 'https://teams.microsoft.com/l/bubble/' + this.AppId + '?url=' + this.baseUrl + '&height=270&width=300&title=ContentBubbleinTeams&completionBotId=' + this.AppId
+      }
+    };
+    await context.sendActivity(replyActivity);
+  }
+
+  // Create adaptive card for sending initial agenda.
+  createAdaptiveCard() {
+    return CardFactory.adaptiveCard(AdaptiveCard);
+  }
+
+  // Create adaptive card for send list of agenda questions.
+  createQuestionAdaptiveCard(myText) {
+    var templatePayload = templateJson;
+    var template = new ACData.Template(templatePayload);
+    var cardPayload = template.expand({
+      $root: {
+        name: myText
+      }
+    });
+    return CardFactory.adaptiveCard(cardPayload);
```

Comment:
One line space before it.

Reply:
This sample app shows how to send in meeting targeted notifications.